hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableAccountStoreImpl.java

```diff
@@ -133,7 +135,37 @@ public boolean contains(@NonNull final AccountID accountID) {
     protected Account getAccountLeaf(@NonNull final AccountID id) {
         // The Account ID may be aliased, in which case we need to convert it to a number-based account ID first.
```

Comment:
Remove this comment as it has changed.

Reply:
Yes Iris. Thanks Will update in next PR.